deepmd/descriptor/se_a_v2.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import logging
+from typing import (
+    List,
+    Optional,
+)
+
+from deepmd.utils.spin import (
+    Spin,
+)
+
+from .descriptor import (
+    Descriptor,
+)
+from .se_a import (
+    DescrptSeA,
+)
+
+log = logging.getLogger(__name__)
+
+
+@Descriptor.register("se_e2_a_v2")
+class DescrptSeAV2(DescrptSeA):
```

Comment:
Does it force one to enable type embedding? If so, I think a better name might be `se_e2_a_tebd_v2` or `se_a_tpe_v2` (there is `se_a_tpe` in `se_a_ebd.py`) since `se_e2_a` without type embedding should not use this type.

The example and documentation should be updated to the new type.